Sources/StreamChatSwiftUI/ChatChannel/MessageList/MessageView.swift

```diff
@@ -228,13 +228,19 @@ struct StreamTextView: View {
     
     @Injected(\.fonts) var fonts
     
-    var message: ChatMessage
+    let message: ChatMessage
+    private let adjustedText: String
+    
+    init(message: ChatMessage) {
+        self.message = message
+        adjustedText = message.adjustedText
+    }
```

Comment:
I don't know why exactly but SwiftUI does not redisplay the view when `message.adjustedText` changes. Did a workaround fix here. 
Case of editing an existing message.